setup/_metapackage/setup.py

```diff
@@ -1,20 +1,20 @@
 import setuptools
 
-with open('VERSION.txt', 'r') as f:
+with open("VERSION.txt", "r") as f:
     version = f.read().strip()
 
 setuptools.setup(
     name="odoo14-addons-oca-mis-builder",
     description="Meta package for oca-mis-builder Odoo addons",
     version=version,
     install_requires=[
-        'odoo14-addon-mis_builder',
-        'odoo14-addon-mis_builder_budget',
-        'odoo14-addon-mis_builder_demo',
+        "odoo14-addon-mis_builder",
+        "odoo14-addon-mis_builder_budget",
+        "odoo14-addon-mis_builder_demo",
     ],
     classifiers=[
-        'Programming Language :: Python',
-        'Framework :: Odoo',
-        'Framework :: Odoo :: 14.0',
-    ]
+        "Programming Language :: Python",
+        "Framework :: Odoo",
+        "Framework :: Odoo :: 14.0",
+    ],
 )
```

Comment:
You should not touch this `setup/` directory.

Reply:
Oupsi, had a problem running black and linting tools, I will check my diff